.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: 🐞 Bug report
+about: Create a report to help us improve node-fetch
+title: "[BUG] "
+labels: bug
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2. 
+3. 
+4. 
+
+**Expected behavior**
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
```

Comment:
Should there be a separate issue template for changes/improvements?

Reply:
check out #816